bennettbot/config.py

```diff
@@ -0,0 +1,39 @@
+import re
+
+from . import settings
+
+
+def get_support_config(channels=None):
+    channels = channels or {}
+    return {
+        "tech-support": {
+            "keyword": "tech-support",
+            # Use the support setting to get the channel. We use channel ID if it is
+            # available, otherwise we default to using the setting value (on the
+            # assumption that the setting value is the channel name).
```

Comment:
Looks good! I guess if we do switch to using channel IDs in the settings, we will have to update this comment in that PR?

Reply:
No - although I think I haven't said it very clearly, and I think I actually meant we assume the setting value is already the channel ID (not name). Currently the setting can be either the name or the ID; we first assume the setting is a channel name, and use it to retrieve the ID from the channels if the channels dict is available.
In the bot case, if if it can't be retrieved, we assume it's the ID already, and code in the handler registration checks that the IDs are valid.  
In the dispatcher case, we don't have the dict of channel names to IDs available, so we default to using whatever the setting is, which may be a name or an ID.

I'll reword this comment